include/firrtl.xml

```diff
@@ -36,6 +36,7 @@
       <item>Reset</item>
       <item>AsyncReset</item>
       <item>Integer</item>
+      <item>List</item>
```

Comment:
Since it's parameterized, and looking at my draft List spec commits (apparently never submitted oops), I think this might render better if you put this down in `outertype`.

IIRC making this work with, e.g., `List<List<Integer>>` was a bear and lead to a bunch of fixes since unlike probes these can nest.

Reply:
Testing with the above nested List indeed this renders better -- in particular the angle brackets and types are consistently highlighted.

Reply:
Thanks, yep that looks good.